assets/css/framework.css

```diff
@@ -0,0 +1,243 @@
+@import url(reset.css);
+
+
+.row {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr); }
+
+.col-1 {
+  grid-column: span 1; }
+
+.col-2 {
+  grid-column: span 2; }
+
+.col-3 {
+  grid-column: span 3; }
+
+.col-4 {
+  grid-column: span 4; }
+
+.col-5 {
+  grid-column: span 5; }
+
+.col-6 {
+  grid-column: span 6; }
+
+.col-7 {
+  grid-column: span 7; }
+
+.col-8 {
+  grid-column: span 8; }
+
+.col-9 {
+  grid-column: span 9; }
+
+.col-10 {
+  grid-column: span 10; }
+
+.col-11 {
+  grid-column: span 11; }
+
+.col-12 {
+  grid-column: span 12; }
+
+body {
+  font-family: 'Roboto Condensed'; }
+
+.text-center {
+  text-align: center; }
+
+.text-left {
+  text-align: left; }
+
+.text-right {
+  text-align: right; }
+
+h1, h2, h3 {
+  font-size: 10px; }
```

Comment:
Hey, wondering why you made these 3 header tags the same font-size, they should be different.
Also, why so small? Is it just me or is it a bit small for a header tag? 😅